crates/uv-resolver/src/resolution.rs

```diff
@@ -453,7 +455,14 @@ impl ResolutionGraph {
                     dist.version_id()
                 )
             };
-            for req in manifest.apply(&archive.metadata.requires_dist) {
+            let uv_requirements: Vec<_> = archive
```

Comment:
Can we call this `requirements`? Again, I'd like to avoid encoding `uv_` in various places.